deepinv/loss/score.py

```diff
@@ -1,41 +1,161 @@
 import torch
-import numpy as np
+import deepinv.physics
 from deepinv.loss.loss import Loss
 
 
 class ScoreLoss(Loss):
     r"""
-    Approximates score of a distribution.
+    Learns score of noise distribution.
 
-    It approximates the score of the measurement distribution :math:`S(y)\approx \nabla \log p(y)`
+    Approximates the score of the measurement distribution :math:`S(y)\approx \nabla \log p(y)`
     https://proceedings.neurips.cc/paper_files/paper/2021/file/077b83af57538aa183971a2fe0971ec1-Paper.pdf.
 
     The score loss is defined as
 
     .. math::
 
-        \| \epsilon - \sigma S(y+ \sigma \epsilon) \|^2
+        \| \epsilon + \sigma S(y+ \sigma \epsilon) \|^2
```

Comment:
This is the same loss for all noise models ? or is it only valid for Gaussian noise ?